src/test/java/com/conveyal/datatools/manager/models/FeedVersionTest.java

```diff
@@ -106,4 +103,45 @@ void canDetectBlockingErrorTypesForPublishing(NewGTFSErrorType errorType) throws
 
         assertThat(feedVersion1.hasBlockingIssuesForPublishing(), equalTo(true));
     }
+
+    /**
+     * {@link FeedSource::publishedVersionId} should be unset if it references a feed version being deleted.
+     */
+    @Test
+    void shouldDeletePublishedVersionIdWhenDeletingVersion() {
```

Comment:
I'm not sure if this is an OS thing, but the feed versions are not deleted after the test has complete from Mongo:
Could not drop feed for namespace published_namespace - ERROR: schema "published_namespace" does not exist.